core/modules/comment/tests/comment.test

```diff
@@ -432,7 +432,7 @@ class CommentInterfaceTest extends CommentHelperCase {
     $this->assertEqual(rtrim($comment_loaded->thread, '/') . '.01/', $reply_loaded->thread, 'Thread of second reply grows correctly.');
     // Test rendered comment for title.
     $this->backdropGet('node/' . $this->node->nid);
-    $this->assertText($title_text, 'Individual comment title not found.');
+    $this->assertText($title_text, 'Individual comment title found.');
```

Comment:
Fixing the message for this assertion - the test that hides comment titles if the one after this, where we set `COMMENT_TITLE_HIDDEN `. What we are testing here is that comments ARE shown (`assertText()` vs `assertNoText()` that's used in the next test).